packages/theme/src/Components/Tabs/index.tsx

```diff
@@ -0,0 +1,162 @@
+import { useWindowSize } from 'react-use'
+import { RoundTab } from './RoundTab'
+
+type MaskTabVariant = 'base' | 'flexible' | 'round'
+
+export interface MaskTabListProps
+    extends React.PropsWithChildren<Pick<ButtonGroupProps, 'classes' | 'disabled' | 'fullWidth' | 'size'>> {
+    onChange(event: object, value: string): void
+    'aria-label': string
+    variant?: MaskTabVariant
+}
+
+const ArrowButtonWrap = styled(Button)(({ theme }) => ({
+    position: 'absolute',
+    top: 0,
+    right: 0,
+    padding: theme.spacing(1.5),
+    height: 38,
+    width: 38,
+    background: '#F2F6FA',
+
+    '&:hover': {
+        background: '#F2F6FA',
+    },
+}))
+
+const ArrowBackIosNewIconWrap = styled(ArrowBackIosNewIcon)(({ theme }) => ({
+    color: theme.palette.text.primary,
+}))
+
+const ButtonGroupWrap = styled(ButtonGroup, {
+    shouldForwardProp: (prop) => prop !== 'maskVariant' && prop !== 'isOpen' && prop !== 'isOverflow',
+})<{ maskVariant?: MaskTabVariant; isOpen: boolean; isOverflow: boolean }>(
+    ({ theme, maskVariant, isOpen, isOverflow }) => ({
+        position: 'relative',
+        display: 'flex',
+        alignItems: 'center',
+        flexWrap: maskVariant === 'flexible' && isOpen ? 'wrap' : 'nowrap',
+        overflow: 'hidden',
+        paddingLeft: theme.spacing(2),
+        paddingRight: theme.spacing(2),
+        flex: 1,
+        transform: '',
+        background:
+            maskVariant === 'flexible' && !isOpen && isOverflow
+                ? 'linear-gradient(270deg, rgba(223, 229, 244, 0.8) 36px, rgba(244, 247, 254, 0) 20%)'
+                : 'transparent',
+    }),
+)
+
+/**
+ * This component is like TabList + Tabs in the @mui/material.
+ * It should be used with <ButtonGroupTab>.
+ *
+ * Warning: Only a few "value" and "label" props on the @mui/material <Tab> component will work.
+ *
+ * @example
+ *  const [currentTab, onChange, tabs, setTab] = useTab('tab1', 'tab2', 'tab3')
+ *  return (
+ *      <TabContext value={currentTab}>
+ *          <MaskTabList onChange={onChange}>
+ *              <Tab label="Item One" value={tabs.tab1} />
+ *              <Tab label="Item Two" value={tabs.tab2} />
+ *              <Tab label="Item Three" value={tabs.tab3} />
+ *          </MaskTabList>
+ *          <TabPanel value={tabs.tab1}>Item One</TabPanel>
+ *          <TabPanel value={tabs.tab2}>Item Two</TabPanel>
+ *          <TabPanel value={tabs.tab3}>Item Three</TabPanel>
+ *      </TabContext>
+ *  )
+ */
+export const MaskTabList = forwardRef<HTMLDivElement, MaskTabListProps>((props, ref) => {
+    const context = useTabContext()
+    const [open, handleToggle] = useState<boolean>(false)
+    const [isTabsOverflow, setTabsOverflow] = useState<boolean>(false)
+    const innerRef = useRef<HTMLDivElement>(null)
+    const { width } = useWindowSize()
+
+    if (context === null) throw new TypeError('No TabContext provided')
```

Comment:
Don't throw error here.

Reply:
I think is ok, Because the Tabs mush be used in TabContext.

Reply:
It's ok